applications/sandbox/systems/testsystem.cpp

```diff
@@ -1,9 +1,39 @@
 #include "testsystem.hpp"
+
+//#define LLRI_ENABLE_VALIDATION 0 //uncommenting this disables internal validation (see docs)
+//#define LLRI_ENABLE_INTERNAL_API_MESSAGE_POLLING 0 //uncommenting this disables internal API message polling
```

Comment:
The primary reason that I used 0/1 flags was because I wanted to have a place where I could document the define's existence/behaviour (which I could do in the #ifdef). I'm open to changing this though, if we are to go with if defined(), where should I document their behaviour? See llri.hpp for their current documentation

Reply:
well although we document using doxygen, we can also add additional pages with defines manually through rst files.
Alternatively what you could do is this:
```cpp
#if defined(DOXY_EXCLUDE)
     /**@def ENABLE_SOMETHING
     * @brief Enables something if defined, disables when undefined.
     */
#   define ENABLE_SOMETHING
#endif

Reply:
that approach seems ok I'll go with it!

Reply:
I decided to go with DISABLE_SOMETHING instead because I wanted these flags to be enabled by default